### DISCUSSION TITLE
Conference use case?

### DISCUSSION THREAD

**Discussion by pietroppeter** [squidfunk/mkdocs-material#6382] [General]
Hi, does anybody know if there are examples around of using Material for MkDocs for a small conference (1-day) website?
I know it is not the intended use case, but I do like the template and I was wondering if there were examples around for inspiration.

It would need to support things like a landing page, a page for conference schedule, a page for venue and logistics. No need for advanced features like tickets (we will likely use eventbrite) nor CfP (the idea is to have all invited speakers), so nothing server side would be required.
Thanks, Pietro

**Comment by alexvoss** [comment]
Have a look on Google for `"made with material for mkdocs" conference` and you will find some hits. I tried my default search engine DuckDuckGo first and did not find them, so perhaps using different search engines and different words instead of "conference" might help. Would love to hear about your experiences and any requirements that Material for MkDocs does not fulfill yet for this use case. Ping me here if you come across any.

Edit: the [PyTexas Conference](https://www.pytexas.org/2024/) is one example.

**Reply by pietroppeter** [reply]
Thanks! Sometimes the answer is just the right way to search for it. :) I will try and report back on my experiences

**Reply by pietroppeter** [reply]
just to close the loop, in the end I received support from PyData community and we are working with adapting the site of PyData Amsterdam (the idea is to try to make it a good template for further pydata conferences). I will look forward though to another opportunity to use this great project. (link to source of site for the curious ones, still very much a WIP and for further distribution ;) https://github.com/pietroppeter/py4ai)
